recipes/sqlite3/all/test_package/CMakeLists.txt

```diff
@@ -5,6 +5,10 @@ include(${CMAKE_BINARY_DIR}/conanbuildinfo.cmake)
 conan_basic_setup()
 
 find_package(SQLite3 REQUIRED)
+message(STATUS "SQLite3_INCLUDE_DIRS: ${SQLite3_INCLUDE_DIRS}")
```

Comment:
is it necessary? might be noizy

Reply:
Not really, it was a test, I can remove those lines.